enacts/wat_bal/layout_monit.py

```diff
@@ -311,6 +328,50 @@ def controls_layout(lat_min, lat_max, lon_min, lon_max, lat_label, lon_label):
                         ),
                         dbc.Button(id="submit_kc", children='Submit'),
                     ),
+                    Block(
+                        "Compare to...",
+                        Sentence(
+                            "Planting Date",
+                            DateNoYear("planting2_", 1, CONFIG["planting_month"]),
+                        ),
+                        Sentence(
+                            Number("year_ago", 1, min=0, max=99, html_size=3),
```

Comment:
Wouldn't it be less confusing to name the years (1998, 1999, ...) instead of counting how far back they are?

Reply:
you mean confusing for the programmer or for the user? I agree for the user it would be likely easier to enter day, month and year.

Either way, the difficulty with having the year as input is the realignment on the same year for plotting purposes...

Let's say the planting dates are 15 Apr 2023 and 3 Apr 2012, then it's easy to rewrite the ts starting in 3 Apr 2012 to start in 3 Apr 2023. But now let's say we have 3 Jan 2023 and 28 Dec 2012. How do I systematically make sure that I re-align on 28 Dec 2022?  I could take all the 28 Dec, take the difference with 3 Jan 2023, and pick the smallest.

Other idea?

Reply:
I meant for the user. Leave the value as integer number of years, but display it as "1 year ago (28 Dec 2022)"?

Reply:
The user now inputs a full planting date (d-m-y) to compare to.